ui/plugins/failure_summary/controller.js

```diff
@@ -63,7 +63,22 @@ treeherder.controller('BugsPluginCtrl', [
                             );
                             suggestions.push(suggestion);
                         });
+                        var errors = [];
+                        if (suggestions.length === 0 && artifact_list.length > 1) {
+                            var artifact = artifact_list[1];
+                            console.log(artifact);
```

Comment:
we try not to leave ``console.log`` messages in the code.  we shoudl take this out after debugging is done